spec/error_reporters/rollbar_spec.rb

```diff
@@ -41,6 +41,10 @@
     context "given an empty rack_env" do
       let(:rack_env) { {} }
 
+      it "expects rack_env to be a hash" do
+        assert_kind_of(Hash, rack_env)
```

Comment:
Is this test needed? It looks like it is just asserting that L42 is a Hash which seems redundant.